documentation/migration-guide-fcm.md

```diff
@@ -0,0 +1,17 @@
+### FCM Migration
+
+FCM migration is optional, but recommended as Google deprecated GCM and is planning on dropping
+GCM client APIs in April of 2019. For more info, please read the [GCM F.A.Q](https://developers.google.com/cloud-messaging/faq).
+
+Urban Airship supports the FCM client library starting in SDK 9.1.0. It is available in a new package
+`urbanairship.android.fcm`. The FCM package is not compatible with the deprecated package `UrbanAirship`.
+
+Migration Steps:
+
+- [Add Firebase to your app](https://firebase.google.com/docs/android/setup#add_firebase_to_your_app).
```

Comment:
Lets avoid their guide

Reply:
@rlepinski: I got that from our Android SDK FCM Migration Guide. Should I just point there instead?

Reply:
My bad, i though it was the bad xamarin guide